test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6266,7 +6295,7 @@ def submit(self, obj, script=None, extend=None, submit_dir=None):
             return None
 
         if submit_dir is None:
-            submit_dir = tempfile.gettempdir()
+            submit_dir = pwd.getpwnam(obj.username)[5]
```

Comment:
This is changing PTL for non-cray to submit from the user's homedir.  Is this desired?